package.json

```diff
@@ -39,12 +40,12 @@
   "devDependencies": {
     "babel-cli": "^6.9.0",
     "babel-core": "^6.2.1",
-    "babel-eslint": "^6.0.4",
+    "babel-eslint": "^6.*.*",
```

Comment:
I thought the `*` was sort of implied when using [Caret Ranges](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).
So `^6.0.0` means the same as `6.*.*`, but `^6.0.4` means you'll accept anything from version 6 as long as it's >=6.0.4.